tests/pytorch/nightly/fs-transformer.libsonnet

```diff
@@ -189,13 +190,23 @@ local utils = import 'templates/utils.libsonnet';
   local tpuVm = common.PyTorchTpuVmMixin {
     tpuSettings+: {
       tpuVmExtraSetup: |||
+        pip3 install tqdm
         git clone --recursive https://github.com/pytorch-tpu/examples.git tpu-examples/
+        git clone -b tpu --single-branch https://github.com/darisoy/fairseq.git fairseq-pjrt/
```

Comment:
You should make these changes in `pjrt` since we don't need to check out the PJRT branch except for the PJRT test. Either append the two new statements like `tpuSettings+: tpuVmExtraSetup+: ||| ... ` or override `tpuVmExtraSetup` like `tpuSettings+: tpuVmExtraSetup: ||| ...`.